domain.yml

```
@@ -46,78 +46,308 @@ slots:
+      type: from_entity
+      conditions:
+      - active_loop: cc_payment_form
+        requested_slot: amount-of-money
+      - active_loop: transfer_money_form
+        requested_slot: amount-of-money
+    - entity: number
+      not_intent:
+      - check_balance
+      - check_earnings
+      type: from_entity
+      conditions:
+      - active_loop: cc_payment_form
+        requested_slot: amount-of-money
+      - active_loop: transfer_money_form
+        requested_slot: amount-of-money
+    - intent:
+      - inform
+      - cc_payment_form
+      type: from_text
+      conditions:
+      - active_loop: cc_payment_form
+        requested_slot: amount-of-money
+    - intent:
+      - inform
+      - transfer_money_form
+      type: from_text
+      conditions:
+      - active_loop: transfer_money_form
+        requested_slot: amount-of-money
+    - type: from_entity
+      entity: amount-of-money
   amount_transferred:
     type: any
     initial_value: 0
     influence_conversation: false
+    mappings:
+    - type: custom
   credit_card:
     type: any
     influence_conversation: false
+    mappings:
+    - entity: credit_card
+      type: from_entity
+      conditions:
+      - active_loop: cc_payment_form
+        requested_slot: credit_card
+    - intent:
+      - inform
+      - cc_payment_form
+      type: from_text
+      conditions:
+      - active_loop: cc_payment_form
+        requested_slot: credit_card
+    - type: from_entity
+      entity: credit_card
   currency:
     type: any
     influence_conversation: false
-    initial_value: "$"
+    initial_value: $
+    mappings:
+    - type: custom
   end_time:
     type: any
     influence_conversation: false
+    mappings:
+    - type: custom
   end_time_formatted:
     type: any
     influence_conversation: false
+    mappings:
+    - type: custom
   grain:
     type: any
     influence_conversation: false
+    mappings:
+    - type: custom
   handoff_to:
     type: any
     influence_conversation: false
+    mappings:
+    - type: from_entity
+      entity: handoff_to
   next_form_name:
     type: text
     influence_conversation: true
+    mappings:
+    - type: custom
   number:
     type: any
     influence_conversation: false
+    mappings:
+    - type: from_entity
+      entity: number
   payment_amount_type:
     type: any
     initial_value: ''
     influence_conversation: false
+    mappings:
+    - type: custom
   previous_form_name:
     type: text
     influence_conversation: true
+    mappings:
+    - type: custom
   repeated_validation_failures:
     type: any
     influence_conversation: false
+    mappings:
+    - type: custom
   requested_slot:
     type: any
     influence_conversation: false
+    mappings:
+    - type: custom
   search_type:
     type: any
     influence_conversation: false
+    mappings:
+    - intent: search_transactions
+      type: from_trigger_intent
+      value: spend
+      conditions:
+      - active_loop: transaction_search_form
```

```suggestion
      type: from_trigger_intent
      value: spend
```
from_trigger mappings have  to be non-conditional

---

The changes are auto-generated by the script we've shipped with global slot mappings. Is this wrong/will break the bot?

---

Most of it is right but in this case it will break the bot  in the sense that this mapping will never be used - there's a blurb in the docs about the need to use unconditionals for `from_trigger_intent`